test/FlatLaunchpeg.test.ts

```diff
@@ -179,6 +194,88 @@ describe('FlatLaunchpeg', () => {
     })
   })
 
+  describe('Pre-mint phase', () => {
```

Comment:
it might be a good idea to dedup our tests in another PR and move them into a separate file (I know the funds flow tests are also in both files, and probably other base launchpeg stuff)

also might want to move the batch reveal tests into a separate file after we merged all our PRs

Reply:
ok sure, will do so. are you proposing to have say a `BaseLaunchpeg.test.ts` file for the common tests? i will review the test coverage when all the PRs are merged and re-factor the tests accordingly.

Reply:
yeah I think that would make sense 🤔

and `BatchReveal.test.ts` eventually to cover the batch reveal logic

Reply:
ok will do